integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/metadata/EventConverter.java

```diff
@@ -0,0 +1,100 @@
+package se.inera.intyg.minaintyg.integration.webcert.converter.metadata;
+
+import static java.util.function.Predicate.not;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import org.springframework.stereotype.Component;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.common.CertificateEvent;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.CertificateMetadataDTO;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.metadata.CertificateRelation;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.metadata.CertificateRelationType;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.metadata.CertificateStatus;
+
+@Component
+public class EventConverter {
+
+  public static final String EVENT_SENT_TO_DESCRIPTION = "Skickat till ";
+  public static final String EVENT_REPLACED = "Ersatt av vården med ett nytt intyg";
+  public static final String EVENT_REPLACES = "Ersätter ett intyg som inte längre är aktuellt";
+
+  public List<CertificateEvent> convert(CertificateMetadataDTO metadataDTO) {
+    final var events = new ArrayList<CertificateEvent>();
+
+    if (metadataDTO.isSent()) {
```

Comment:
Check if there is a recipient and then if it has been sent... The isSent() will later be removed from the api..